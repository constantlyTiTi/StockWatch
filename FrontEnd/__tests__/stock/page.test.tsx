import { render, screen } from "@testing-library/react";
import Page from "@/stock/page"
import {describe, expect, it, jest} from '@jest/globals'

jest.mock("@/components/stock/StockListTable",() => "StockList")


describe("Stock List Page ", () => {


    it("stock Home Page should render a stock List table", () => {
        const {container} = render(<Page/>);

        expect(container).toMatchSnapshot()

    })
})