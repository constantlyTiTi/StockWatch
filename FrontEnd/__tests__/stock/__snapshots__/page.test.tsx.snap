// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Stock List Page  stock Home Page should render a stock List table 1`] = `
<div>
  <stocklist />
</div>
`;
